package.json

```diff
@@ -70,12 +70,13 @@
     "uglify-js": "^3.1.4",
     "underscore": "^1.5.1",
     "webpack": "^1.12.6",
-    "webpack-dev-server": "^1.12.1"
+    "webpack-dev-server": "^1.12.1",
+    "coffeescript": "^2.6.1",
```

Comment:
Why did you add `coffeescript` as a dependency?

Reply:
```
 ~/source/repos/tmp/aster/node_modules/handlebars  npm run build

> handlebars@5.0.0-alpha.1 build
> grunt build

Loading "bg-shell.coffee" tasks...ERROR
>> Error: Grunt attempted to load a .coffee file but CoffeeScript was not installed.
>> Please run `npm install --dev coffeescript` to enable loading CoffeeScript.
Loading "child-process.coffee" tasks...ERROR
>> Error: Grunt attempted to load a .coffee file but CoffeeScript was not installed.
>> Please run `npm install --dev coffeescript` to enable loading CoffeeScript.

Running "babel:cjs" (babel) task

Running "webpack:handlebars" (webpack) task
Version: webpack 1.15.0
        Asset    Size  Chunks             Chunk Names
handlebars.js  572 kB       0  [emitted]  main

Running "webpack:runtime" (webpack) task
Version: webpack 1.15.0
                Asset    Size  Chunks             Chunk Names
handlebars.runtime.js  383 kB       0  [emitted]  main

Done.
```